app/views/shared/_notifications.html.erb

```diff
@@ -0,0 +1,5 @@
+<% flash.each do |name, msg| -%>
+  <div class="<%= name %>">
```

Comment:
Tailwind would be nice 😄

Reply:
people sound excited about Tailwind so it might be a good idea, but I haven't tried it myself so currently I don't have an opinion. We can discuss it in a separate PR and probably get Bocha's thoughts on it.